package.json

```diff
@@ -27,7 +27,7 @@
     "tsc": "tsc --project ."
   },
   "dependencies": {
-    "@api3/ois": "^2.2.1",
+    "@api3/ois": "file:../ois",
```

Comment:
I will change this when OIS is published after https://github.com/api3dao/ois/pull/117 is merged and new OIS is released.